.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
-  - "4.2"
+  - "4"
```

Comment:
Ehm ... I would very much like to keep `4.2`, as that's the Long Term Support version, so a lot of cautious users will be relying on that.

https://nodejs.org/en/blog/release/v4.2.0/

Perhaps we can have `4` AND `4.2`?